MANIFEST.in

```diff
@@ -4,3 +4,8 @@ include gimel/VERSION
 include gimel/config.json.template
 include requirements.txt
 recursive-include gimel/vendor *
+
+exclude tox.ini
+exclude requirements.in
+exclude requirements.txt
```

Comment:
Currently the setup.py requirements are explicitly defined without `requirements.txt`. See https://github.com/Alephbet/gimel/blob/master/setup.py#L6 - not sure what's the best practice though.